src/AccessToken.php

```diff
@@ -317,4 +425,32 @@ protected function callJwtStatic($method, array $args = [])
             : 'JWT';
         return call_user_func_array([$class, $method], $args);
     }
+
+    /**
+     * Provide a hook to mock calls to the JWT static methods.
+     *
+     * @param string $method
```

Comment:
This is unused

Reply:
What do you mean? It's used in the function:

```php
    protected function callJwtStatic($method, array $args = [])
    {
        $class = class_exists('Firebase\JWT\JWT')
            ? 'Firebase\JWT\JWT'
            : 'JWT';
        return call_user_func_array([$class, $method], $args);
    }
```

Reply:
Well, I just noticed it isn't used in the signature of this protected method here